SUPPORTED.md

```diff
@@ -0,0 +1,86 @@
+# SQL Supported Commands
+
+## Comparisson expressions
+- !=
+- ==
+- \>
+- <
+- \>=
+- <=
+- BETWEEN
+- IN
+- NOT IN
+- REGEXP
+
+## Null check expressions
+- IS NOT NULL
+- IS NULL
+
+## Grouping expressions
+- AVG
+- COUNT
+- MAX
+- MIN
+
+## Standard expressions
+- ALIAS (AS)
+- LITERAL
+- STAR (*)
+- CAST/CONVERT
+- CREATE TABLE
+- DESCRIBE/DESC/EXPLAIN FORMAT=TREE
```

Comment:
FORMAT=TREE is only needed on `EXPLAIN [query]`
Maybe we should split this into two commands, `DESCRIBE/DESC/EXPLAIN [tablename]` and `DESCRIBE/DESC/EXPLAIN FORMAT=TREE [query]`

Reply:
done